js/components/charts/boxplot.js

```diff
@@ -1,25 +1,25 @@
 define([
 	'knockout',
   'providers/Chart',
+  'providers/Component',  
   'atlascharts',
+  'text!components/charts/chart.html',
+  'const'
 ], function (
   ko,
   Chart,
+  Component,
   atlascharts,
+  view,
+  helpers
 ) {
   class Boxplot extends Chart {
-    constructor() {
-      super();
-      this.name = 'boxplot';
-      this.chart = new atlascharts.boxplot();
+    constructor(params) {
+      super(params);
+      this.renderer = new atlascharts.boxplot();
     }
 
-    render(params) {
-      super.render(params);
-      return this;
-    }
   }
 
-  const viewModel = new Boxplot();  
-	return viewModel.build();
+	return helpers.build(Boxplot, 'boxplot', view);
```

Comment:
Use destructuring. Put component's name first. Use original KO's keys for `viewModel` and `template`